api/wasm/cpp/Makefile.base

```diff
@@ -0,0 +1,8 @@
+API:=$(shell git rev-parse --show-toplevel)/api/wasm/cpp
```

Comment:
Seems only reasonable if user use git.  I am fine with current phase

Reply:
I imagine that users will have to changes this file or link to the api in their project.